src/help/zaphelp/contents/ui/dialogs/options/certificate.html

```diff
@@ -7,12 +7,26 @@
 </TITLE>
 </HEAD>
 <BODY BGCOLOR="#ffffff">
-<H1>Options Certificate screen</H1>
+<H1>Options Client Certificate screen</H1>
 <p>
-The Certificate screen allows you to add a client certificate for ZAP to use when testing applications
+The Client Certificate screen allows you to add a client certificate for ZAP to use when testing applications
 protected using mutual SSL.
 </p>
-
+It is also possible to set the Client Certificate from the commandline with the following switches:
+<pre>
+-config certificate.use=true
+-config certificate.pkcs12.path=/path/to/file.p12
+-config certificate.pkcs12.password=WhateverThePasswordIs
+-config certificate.pkcs12.index=1
+-config certificate.persist=1
+</pre>
+<p>
+<TT>-config certificate.pkcs12.index</TT> is only necessary if the file contains multiple certificates 
```

Comment:
I'm not sure if using `tt` tags is the best option here, I'm open to suggestions.

Reply:
I'd suggest using code tags instead, that would have different rendering in the wiki page.